tests/test_eigen_tensor.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+
+np = pytest.importorskip("numpy")
+m = pytest.importorskip("pybind11_tests.eigen_tensor")
+
+tensor_ref = np.array(
+    [
+        [[0.0, 15.0], [3.0, 18.0], [6.0, 21.0], [9.0, 24.0], [12.0, 27.0]],
```

Comment:
Better use random tensors, e.g., generated using `numpy.random.rand`? No need to hard code the coefficients.

Reply:
These are explicitly to ensure that indexing is the same on both the C++ and Python side. I updated the test to make it more obvious what those coefficients mean.

Reply:
Use an integer tensor then?